src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/TargetCompatibilityReferenceAnalyzer.cs

```diff
@@ -67,21 +68,18 @@ public async Task<PackageAnalysisState> AnalyzeAsync(IProject project, PackageAn
                         if (isMajorChange)
                         {
                             _logger.LogWarning("Package {NuGetPackage} has been upgraded across major versions ({OldVersion} -> {NewVersion}) which may introduce breaking changes", packageReference.Name, packageReference.Version, updatedReference.Version);
-                            state.PossibleBreakingChangeRecommended = true;
                         }
 
                         if (updatedReference.IsPrerelease)
                         {
                             _logger.LogWarning("Package {NuGetPackage} has been upgraded to a prerelease version ({NewVersion}) because no released version supports target(s) {TFM}", packageReference.Name, updatedReference.Version, string.Join(", ", project.TargetFrameworks));
                         }
 
-                        state.PackagesToRemove.Add(packageReference);
-                        state.PackagesToAdd.Add(updatedReference);
+                        state.Packages.Remove(packageReference);
+                        state.Packages.Add(updatedReference, isMajorChange ? BuildBreakRisk.Medium : BuildBreakRisk.None);
```

Comment:
`BuildBreakRisk.None` or `BuildBreakRisk.Low`? It's a question because I'm not really sure :) You would *hope* that minor version NuGet package updates would not introduce build break risk but I'm not sure if that's none or low.

Reply:
Oh good point. I like putting it at low since it's probably a non-zero risk